grpc/dataplane_grpc.proto

```diff
@@ -0,0 +1,90 @@
+syntax = "proto3";
+
+package config;
+
+option go_package = "dataplane/grpc";
+
+service ConfigService {
+  rpc GetConfig (GetConfigRequest) returns (GlobalConfig);
+  rpc GetConfigGeneration (GetConfigGenerationRequest) returns (GetConfigGenerationResponse);
+  rpc UpdateConfig (UpdateConfigRequest) returns (UpdateConfigResponse);
+}
+
+message GetConfigRequest {}
+
+message UpdateConfigRequest {
+  GlobalConfig config = 1;
+}
+
+message UpdateConfigResponse {
+  Error error = 1;
+  string message = 2;
+}
+
+enum Error {
+  ERROR_NONE = 0;
+  ERROR_VALIDATION_FAILED = 1;
+  ERROR_APPLY_FAILED = 2;
+  ERROR_UNKNOWN_ERROR = 3;
+}
+
+enum IfType {
+  IF_TYPE_UPLINK = 0;
+  IF_TYPE_MANAGEMENT = 1;
+  IF_TYPE_VXLAN = 2;
+}
+
+message CheckConfigGenerationRequest {}
+
+message CheckConfigGenerationResponse {
+  uint64 generation = 1;
+}
+
+message Subnet {
+  string cidr = 1;
+  string name = 2;
+  uint32 vni = 3;
```

Comment:
I don't think this VNI is needed?

Reply:
Does the VPC VNI is the only valuable for us?